astropy/nddata/tests/test_nddata.py

```diff
@@ -439,9 +440,6 @@ def test_pickle_nddata_without_uncertainty():
 # Check that the meta descriptor is working as expected. The MetaBaseTest class
 # takes care of defining all the tests, and we simply have to define the class
 # and any minimal set of args to pass.
```

Comment:
This comment needs to be moved also

Reply:
I initially thought that too but I'm actually not sure. It seems to me that it should be attached to the `TestMetaNDData` class rather than the import statement ?

Reply:
The comment explains what `MetaBaseTest` does and it is placed where it is used, so I don't see why we would want to move it.